src/libraries/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -756,21 +756,14 @@ public unsafe void CopyTo(Array array, int index)
 
             if (array is int[] intArray)
             {
-                Div32Rem(m_length, out int extraBits);
+                int quotient = Div32Rem(m_length, out int extraBits);
 
-                if (extraBits == 0)
-                {
-                    // we have perfect bit alignment, no need to sanitize, just copy
-                    Array.Copy(m_array, 0, intArray, index, m_array.Length);
```

Comment:
I think the only change necessary was to replace `m_array.Length` with `m_length`. Am I missing something?

Reply:
Yes, m_length is the number of bits stored the `BitArray` represents, so that's why we need to divrem by 32 to get extraBits and quotient (number of full ints)

Reply:
Got it, thanks.